.github/workflows/main.yml

```diff
@@ -95,9 +95,9 @@ jobs:
       if: steps.changes.outputs.docker == 'true'
       uses: docker/build-push-action@v4
       with:
-        context: .
+        context: ./docker
```

Comment:
is Dockerfile moving to ./docker folder with this pr?

Reply:
I just tried changing the context from . to ./docker - because  before that it was failing with "Cannot find Dockerfile"
Now the tests are failing where they were passing before.  I guess because before it was building a Docker test image, now it is trying to pull it from Dockerhub, but we haven't successfully pushed to Dockerhub since I changed the requirements (to SQLAlchemy 2.0)

N.B. Dockerfile has ALWAYS lived in ./docker (assuming . is the root directory of the repository.)

Reply:
https://github.com/docker/build-push-action#git-context

```
   -
        # Setting up Docker Buildx with docker-container driver is required
        # at the moment to be able to use a subdirectory with Git context
        name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v2
      -
        name: Build and push
        uses: docker/build-push-action@v4
        with:
          context: "{{defaultContext}}:mysubdir"
          push: true
          tags: user/app:latest
```